src/docs/PCW/PCWUserGuide.adoc

```diff
@@ -49,7 +49,7 @@ Another type of monitoring available in PCW is to check host accessibility. The
 The core of rule processing is managed by the https://www.drools.org/[Drools Engine^] for performance processing of collected metrics. It features powerful mechanisms for complex event processing. The advanced Drools algorithm helps fast detection of fulfilling the conditions and triggering actions.
 
 [[_pcw_rule_action]]
-=== Rule’s Action
+=== Rule’s action
 According to the concepts of PCW, the action will be triggered when specified condition is met. +
 In order to make an action flexible to any user needs, the rule’s action is firing a job inside scheduler. A user just need to specify which workflow from Catalog will be fired as Rule’s action. +
```

Comment:
@ShatalovYaroslav about the "rule" capitalization, you are right that most are lowercase now, my bad, I got mixed up.
Here is an occurrence that should be set to lowercase then.

Reply:
@luispinedamx Yes, thanks. I'm fixing it.